src/test/integration/stacks/workspace/api-gateway/main.tf

```diff
@@ -0,0 +1,67 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
```

Comment:
We have an ignore for files inside workspace folders for copywrite, why do we want them here?

Reply:
They are in here because I copied the files as is from the stacks example for easier diffing in the future

Reply:
roger roger